CPAC/info.py

```diff
@@ -43,8 +43,8 @@
 # version
 _version_major = 1
 _version_minor = 8
-_version_micro = 6
-_version_extra = 'dev'
+_version_micro = 5
+_version_extra = 'dev1'
```

Comment:
I think we want to keep this at
```suggestion
_version_micro = 6
_version_extra = 'dev'
```